backend/tests/test_auth_api.py

```
@@ -1,70 +1,5 @@
-from sqlalchemy.orm import Session
-from app.main import app
-from app.database import get_db
-from app.models import User
-from app.utils import auth_utils  
 
-client = TestClient(app)
-def test_password_hashing():
-    raw_password = "test123"
-    hashed = auth_utils.hash_password(raw_password)
-    assert auth_utils.verify_password(raw_password, hashed)
+def test_basic_math():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_basic_math`

Add return type annotation: `None`

(ANN201)